packages/babel-plugin-proposal-class-properties/test/fixtures/private/tagged-template-static/exec.js

```diff
@@ -0,0 +1,18 @@
+class Foo {
```

Comment:
We don't need to test for private methods here, since we already test them in `packages/babel-plugin-proposal-private-methods/test/fixtures/private-static-method/tagged-template/exec.js`. This test can be simplified to
```js
class Foo {
  static #tag = function () { return this };

  static getReceiver() {
    return this.#tag``;
  }
}

expect(Foo.getReceiver()).toBe(Foo);
```

The same applies for `input.js`, and for `packages/babel-plugin-proposal-class-properties/test/fixtures/private/tagged-template/`.